deploys.go

```diff
@@ -74,6 +74,42 @@ type DeployVersionResult struct {
 	Name string `json:"name"`
 }
 
+// DeployVersionListResult stores a list of deployment versions
+type DeployVersionListResult struct {
+	Versions []*DeployVersionResult `json:"versions"`
+}
+
+func newRespErr(response *http.Response, msg string) error {
+	body := []byte{}
+	response.Body.Read(body)
+	return fmt.Errorf("%s: %s - %q", msg, response.Status, body)
+}
+
+// DeployProjectVersions will list existing versions of a deployment project
+func (d *DeployService) DeployProjectVersions(deploymentProjectID int) ([]*DeployVersionResult, error) {
+	request, err := d.client.NewRequest(http.MethodGet, fmt.Sprintf("deploy/project/%d/versions", deploymentProjectID), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	q := request.URL.Query()
+	// Setting max-results very high to try and get all results
+	q.Set("max-results", "10000")
+	request.URL.RawQuery = q.Encode()
+
+	result := &DeployVersionListResult{}
+	response, err := d.client.Do(request, &result)
+	if err != nil {
+		return nil, err
+	}
+
+	if response.StatusCode != 200 {
```

Comment:
Can you change this, and the other places it is used, to use http.StatusOK? Another thing I have been meaning to do that makes the code a bit more readable. Thank you!